### DISCUSSION TITLE
how to compare across samples, is it already possible?

### DISCUSSION THREAD

**Discussion by BartBryant** [stuart-lab/signac#1669] [Q&A]
goal:  to compare specific cell types across samples

for example, have two samples (wt and mut). I want to compare 'cell-type A' across the samples, wt and mut.

is the code just to follow the 'merging objects' tutorial then use 'FindMarkers(  )' and use arguments 'ident.1' and 'ident.2' to compare? 

also, does one merge 'objects' that were created using the 'analyzing pbmc scatac-seq' with cell types identified with scRNAseq integration? or is this too computationally expensive? 

with the 'combined' seurat object of two samples, how does one find the 'idents' for cell types across samples?

sorry if these are obvious questions, thx for the awesome tool 
AND
thx for keeping it updated/maintained for others to use, these actions are very much appreciated.

barT

W. Bart Bryant Ph.D.

**Comment by BartBryant** [comment]
also, in order to do the 
'Finding co-accessible networks with Cicero'
does the SeuratObject need to run through 'Trajectory Analysis' before one can do the CCAN tutorial?

**Comment by timoast** [comment]
> is the code just to follow the 'merging objects' tutorial then use 'FindMarkers( )' and use arguments 'ident.1' and 'ident.2' to compare?

Yes, to compare across samples you can follow the [merge vignette](https://stuartlab.org/signac/articles/merging) to first create an object containing all of your samples, then after annotating or clustering the cells use the `FindMarkers()` function to compare different groups of cells.

> also, does one merge 'objects' that were created using the 'analyzing pbmc scatac-seq' with cell types identified with scRNAseq integration? or is this too computationally expensive?

To merge objects you will need to ensure that the peaks are the same in all the objects, so we recommend to follow what's shown in the merge vignette.

> with the 'combined' seurat object of two samples, how does one find the 'idents' for cell types across samples?

To annotate cell types you can cluster and look at accessibility at marker genes, or use multimodal label transfer as shown here: https://stuartlab.org/signac/articles/pbmc_vignette#integrating-with-scrna-seq-data

> also, in order to do the
'Finding co-accessible networks with Cicero'
does the SeuratObject need to run through 'Trajectory Analysis' before one can do the CCAN tutorial?

No, these are separate and independent. We just happen to use the same dataset in both the trajectory and Cicero vignettes.

**Comment by BartBryant** [comment]
hi tim,

after the merge vignette, got additional questions with overall goal of comparison of cell types across samples.

after merging objects, i obtained 'gene annotations' and added 'gene.activities' to the SeuratObject following 
https://stuartlab.org/signac/articles/pbmc_vignette#integrating-with-scrna-seq-data

however, when I query the 'merged' object for features, my known cell type markers were quite sparse (more than when either dataset was queried on its own). 
also, when tried 'FindTransferAnchors( )' I got this error message

'Error: No features to use in finding transfer anchors. To troubleshoot, try explicitly 
providing features to the features parameter and ensure that they are present in both reference and query assays.'

Of note, the 'reference' was an integrated scRNAseq dataset as well, does the scRNAseq integration step not allow one to use an integrated object (wt and mut) as a reference?

**Comment by BartBryant** [comment]
-RE:  error in 'FindTransferAnchors(  )', when I used a single scRNA-seq dataset as a 'reference', the fxn worked
but
the 'combined' SeuratObject (wt and mutant) misses a good number of peaks compared to when the scATAC data were queried alone. is this data loss due to different filter functions and creating a common peak set?

another question, can the merge-vignette work with a multi-ome experiment (an h5 file with both ATAC and GTEX modalities)
